library/Mockery/Matcher/MatcherAbstract.php

```diff
@@ -25,7 +25,7 @@ abstract class MatcherAbstract
     /**
      * The expected value (or part thereof)
      *
-     * @var mixed
+     * @var array|mixed[]
      */
     protected $_expected = null;
```

Comment:
This is the opposite from the other two changes because by the time we assign the constructor input to this property, php have already transformed a single or multiple arguments into an array.